mappings/net/minecraft/nbt/NbtCompound.mapping

```diff
@@ -86,6 +94,9 @@ CLASS net/minecraft/class_2487 net/minecraft/nbt/CompoundTag
 		ARG 1 key
 		ARG 2 value
 	METHOD method_10573 contains (Ljava/lang/String;I)Z
+		COMMENT Returns whether the compound object contains an element of the specified type at the specified key.
+		COMMENT
+		COMMENT @return {@code true} if the key exists and the element's type is compatible with {@code type}, else {@code false}
```

Comment:
What is compatibility? The main documentation for this method describes it as 'of the specified type', implying that the type has to exactly match.